packages/twenty-server/src/engine/workspace-manager/workspace-health/interfaces/workspace-table-definition.interface.ts

```diff
@@ -8,6 +8,7 @@ export interface WorkspaceTableStructure {
   isPrimaryKey: boolean;
   isForeignKey: boolean;
   isUnique: boolean;
+  isArray: boolean;
```

Comment:
okay but I feel this should not be part of "WorkspaceTableStructure", it should be in WorkspaceTableColumnStructure (same for dataType, columnName, ...) but I guess it's another story!

Reply:
Actually `WorkspaceTableStructure` is `WorkspaceTableColumnStructure`, we can rename it if you want, but it's only for a given column

Reply:
mmh ok, would be great to rename it then indeed!